packages/less/BUILD.bazel

```diff
@@ -33,6 +33,8 @@ npm_package(
     srcs = [
         "@npm_bazel_less//:package_contents",
     ],
+    # less package is deprecated
+    tags = ["do-not-publish"],
```

Comment:
Update release script to key off of this tag?

Reply:
it already does, that's how we avoided publishing `@bazel/rollup` and `@bazel/terser` to this point.